index.html

```diff
@@ -13424,193 +13424,346 @@ <h2>IANA Considerations</h2>
 
   <p>This section has been submitted to the Internet Engineering Steering
     Group (IESG) for review, approval, and registration with IANA.</p>
+  
+  <section>
+    <h3>application/ld+json</h3>
+    <dl>
+      <dt>Type name:</dt>
+      <dd>application</dd>
+      <dt>Subtype name:</dt>
+      <dd>ld+json</dd>
+      <dt>Required parameters:</dt>
+      <dd>N/A</dd>
+      <dt>Optional parameters:</dt>
+      <dd>
+        <dl>
+          <dt><code>profile</code></dt>
```

Comment:
My suggestion to reduce duplication for the `profile` parameter is to create a separate sub-section for that parameter, and change this to a reference with "See description in <a href="#profile-parameter"></a>", or something similar. That's a bit too much to attempt through the web UI, but I can push a commit to the source branch if this is agreeable.

Reply:
I don't have an opinion on this, related to JSON-LD's use of the media type parameter.

There are other structures suffixes (like +cose) that inherit media type parameters from their normal media type...

cose-type for application/cose and +cose... 

profile for application/ld+json and +ld+json ?

not my call to make... but I agree there should be clarity provided on this topic.

Reply:
go ahead and take over the branch completely, I just came to start the party, you all will probably need to finish it.